deploy/webhook/cloudbuild.yaml

```diff
@@ -0,0 +1,31 @@
+steps:
+  # Get cluster credentials
+- name: gcr.io/k8s-skaffold/skaffold:v0.16.0
+  args: 
```

Comment:
I don think you need this step, cloud build already has the permissions to deploy to clusters in the project

Reply:
when I tried with my own cluster this step was needed for some reason, but am happy to remove it for now and add it back in later if it doesn't work!

Reply:
thats because we've given the cloud build service account cluster admin access explicitly. if you want it to work on you're own cluster in a different project it won't work

Reply:
gotcha, thanks!